app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -41,17 +43,17 @@ public View onCreateView(LayoutInflater inflater,
         setHasOptionsMenu(true);
 
         final View view = inflater.inflate(R.layout.fragment_publications, container, false);
-        mRecyclerView = (RecyclerView) view.findViewById(R.id.publications_rv);
         configureViews();
 
+        GridView gridview = (GridView) view.findViewById(R.id.publicationsGridView);
```

Comment:
Is there a reason we're using a GridView instead of a RecyclerView with a GridLayout Manager? I think its more efficient if we add more publications